builtin/credential/aws-kms/path_config_user.go

```diff
@@ -0,0 +1,64 @@
+package awsKms
+
+import (
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigUser() *framework.Path {
+	return &framework.Path{
+		Pattern: "config/user",
+		Fields: map[string]*framework.FieldSchema{
+			"access_key": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Access key with kms:Decrypt permission.",
+			},
+
+			"secret_key": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Secret key with kms:Decrypt permission.",
+			},
+		},
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.WriteOperation: pathConfigRootWrite,
+		},
+
+		HelpSynopsis:    pathConfigRootHelpSyn,
+		HelpDescription: pathConfigRootHelpDesc,
+	}
+}
+
+func pathConfigRootWrite(
+	req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
```

Comment:
Im not a GO expert but the below code never returns anything for `*logical.Response` so is there is a reason to return it at all?